src/Entity/EntityCustomField.php

```diff
@@ -159,4 +164,20 @@ public static function getStorageFieldName()
     {
         return 'entityValue';
     }
+
+    public function __clone()
+    {
+        $this->id = null;
+        $data = $this->getEntityData();
+        if (is_array($data)) {
+            $newData = new ArrayCollection();
+            foreach ($data as $d) {
+                $newData[] = $d;
+            }
+        } else {
+            $newData = $data;
+        }
+        $this->setValue($newData);
+        $this->setStrRepresentation($this->createStrRepresentation());
```

Comment:
Why do we need to update the strRepresentation? Does it change?

Reply:
nope, does not change. will be removed.